hedera-node/src/main/java/com/hedera/services/ledger/HederaLedger.java

```diff
@@ -141,7 +143,9 @@ public HederaLedger(
 			final SideEffectsTracker sideEffectsTracker,
 			final AccountRecordsHistorian historian,
 			final GlobalDynamicProperties dynamicProperties,
-			final TransactionalLedger<AccountID, AccountProperty, MerkleAccount> accountsLedger
+			final TransactionalLedger<AccountID, AccountProperty, MerkleAccount> accountsLedger,
+			final TransactionalLedger<TokenID, TokenProperty, MerkleToken> tokensLedger,
```

Comment:
We should address @tinker-michaelj 's comment on using `BackingStore` for the tokens instead of `TransactionalLedger`. We do not need a transactional ledger for transfers we only need to "access data"